src/features/node-common.js

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+import _path from 'path';
+import url from 'url';
+
+import fileUrl from 'file-url';
+
+
+export const URL = global.URL
+  ? global.URL
+  : url.URL;
+
+
+export const pathToFileURL = url.pathToFileURL
```

Comment:
This function abstracts away the difference between Node 10+ and earlier versions that worked with URLs differently.